napari/_qt/dialogs/qt_plugin_dialog.py

```diff
@@ -80,26 +101,63 @@ def __init__(
         env.insert(
             "PATH", QProcessEnvironment.systemEnvironment().value("PATH")
         )
-        self.process.setProcessEnvironment(env)
-        self.process.finished.connect(lambda: plugin_manager.discover())
-        self.process.finished.connect(lambda: plugin_manager.prune())
-        self.set_output_widget(output_widget)
+        process.setProcessEnvironment(env)
+        self._processes.append(process)
+        self.set_output_widget(self._output_widget)
+        process.finished.connect(
+            lambda ec, es: self._on_process_finished(process, ec, es)
+        )
+        return process
 
     def set_output_widget(self, output_widget: QTextEdit):
         if output_widget:
             self._output_widget = output_widget
-            self.process.setParent(output_widget)
+            # self.process.setParent(output_widget)
```

Comment:
do you want to either remove this or keep it? =)

Reply:
Fixed!